Mathlib/Data/Real/Archimedean.lean

```diff
@@ -133,6 +133,11 @@ protected theorem is_glb_sInf (S : Set ℝ) (h₁ : S.Nonempty) (h₂ : BddBelow
   exact Real.isLUB_sSup _ h₁.neg h₂.neg
 #align real.is_glb_Inf Real.is_glb_sInf
 
+theorem exists_isGLB (S : Set ℝ) (hne : S.Nonempty) (hbdd : BddBelow S) :
+    ∃ x, IsGLB S x := by
+  use sInf S
```

Comment:
Can you golf this into one line, say (untested) `exact ⟨sInfs, Real.is_glb_sInf S hne hbdd⟩` (and then remove the `by exact`)?